### PR TITLE
Update nextcloud.md

### PR DESCRIPTION
**Pull request by davidsmoot**
Changed the example URL for sync configuration from one that did not work for me to one that did.  

Previous sync URL example did not work and I was unable to locate the "correct" one as recommended through my Nextcloud instance.  

Basing my changes on successful testing and adaptation of the thread at https://www.reddit.com/r/NextCloud/comments/1198e2i/setup_joplin/

Sorry for the vague commit but best I can do at the moment.  Happy to test or help if I can.

### CODE REVIEW

**Comment by davidsmoot** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by laurent22** [comment]
Thanks for the update but if you search online you'll find that sometimes it's `remote.php/dav/files` sometimes it's `remote.php/webdav`. And if I merge this, maybe two years from now someone will create a new pull request with the opposite change

**Comment by davidsmoot** [comment]
I clarified a bit based on your feedback, thank you.

I'm fine with whatever you decide, this has been me learning to do pull requests so no offense taken if you prefer the documentation as is.
But for me, I almost adopted a competitor product that was much less useful / featured / secure because I could not get sync to work.  Clarifying might assist adoption.

File: readme/apps/sync/nextcloud.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
...
+In the **desktop application** or **mobile application**, go to the [Configuration screen](https://github.com/laurent22/joplin/blob/dev/readme/apps/config_screen.md) and select Nextcloud as the synchronisation target. Then input the WebDAV URL (to get it, click on Settings in the bottom left corner of the page, in Nextcloud), this is normally `https://example.com/nextcloud/remote.php/webdav/Joplin` or`https://example.com/nextcloud/remote.php/dav/files/Joplin` on some instances (**make sure to create the "Joplin" directory in Nextcloud**), and set the username and password. If it does not work, please [see this explanation](https://github.com/laurent22/joplin/issues/61#issuecomment-373282608) for more details.  Note that Joplin is simply using the standard WebDav capabilities of NextCloud, if sync does not work then confirm your NextCloud WebDav URL and capabilities with other WebDav applications.
```
Comment (tessus) [inline readme/apps/sync/nextcloud.md]: A space is missing between `or` and the `link`.

Another problem is that `https://example.com/nextcloud/remote.php/dav/files/Joplin` is wrong, unless you are using a user called `Joplin` and the data is kept in the root dir of the user.

It should be: `https://example.com/nextcloud/remote.php/dav/files/<nextcloud-user>/Joplin`

**Comment by davidsmoot** [comment]
Sorry, I should have known better than to try and clean up the formatting at 36,000 feet on a bumpy airplane.  Will clean it up and pay attention to detail when I am in front of my desktop.

**Pull request merged**